protocol-definitions/CPMap.yaml

```diff
@@ -236,3 +236,44 @@ methods:
           since: 2.7
           doc: |
             True if key was associated with newValue, otherwise false.
+  - id: 7
+    name: putIfAbsent
+    since: 2.7
+    doc: |
+      Puts the key-value into the specified map if the key is not currently associated with a value.
+    request:
+      retryable: false
+      partitionIdentifier: -1
```

Comment:
? isn't it partition specific op?

Reply:
As discussed, all CP is same.

Reply:
hmm what does "all cp is same" mean?

Reply:
All CP ops use partition id of `-1`. Same with `CPMap` which is already there and in use.